infrastructure/templates/template.json

```diff
@@ -270,6 +270,20 @@
         "description": "Gov UK Notify service template Id for new release notification"
       }
     },
+    "originalReleasePublicationEmailTemplateId": {
+      "type": "securestring",
+      "defaultValue": "",
+      "metadata": {
+        "description": "Gov UK Notify service template Id for amended release notification"
+      }
+    },
+    "amendedReleasePublicationEmailTemplateId": {
```

Comment:
I think you are missing changes to `template.json` within the config for the Notifier Function app to make use of these parameters. Without that, nothing is going to use them.

See elsewhere for 
`"PublicationNotificationEmailTemplateId": "[parameters('publicationNotificationEmailTemplateId')]",`
 (which will need removing btw).